front/src/common/api/generatedEditoastApi.ts

```diff
@@ -2774,7 +2774,7 @@ export type ScenarioPatchForm = {
   name?: string | null;
   tags?: string[] | null;
 };
-export type RollingStockComfortType = 'STANDARD' | 'AC' | 'HEATING';
+export type RollingStockComfortType = 'STANDARD' | 'AIR_CONDITIONING' | 'HEATING';
```

Comment:
This file cannot be edited by hand. It's automatically generated from OpenAPI, which is auto-generated from the editoast Rust code.

Reply:
Yes, I know, I have to wait until the TSV1 is completely abandoned before I can resume work on this ticket, @younesschrifi is on it